src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.scss

```diff
@@ -0,0 +1,4 @@
+.select {
+  margin-bottom: 15px;
+  max-width: 50%;
```

Comment:
Try to use carbon spacing. And not sure why do we need this ?

Reply:
Yes, not required.